cypress/fixtures/rosa-hosted/RosaClusterHostedWizardValidation.json

```diff
@@ -102,7 +102,7 @@
       "HostPrefix": [
         {
           "Value": "hello",
-          "Error": "The value '/hello' isn't a valid subnet mask. It must follow the RFC-4632 format: '/16'"
+          "Error": "The value 'hello' isn't a valid subnet mask. It must follow the RFC-4632 format: '/16'"
```

Comment:
Could you please take a look at why this change was made? It doesn’t seem correct to me.

Reply:
@akkiangadi @kdoberst FYI ^^